package.json

```diff
@@ -65,6 +65,11 @@
     "whatwg-fetch": "^3.6.20",
     "yaml-lint": "^1.7.0"
   },
+  "overrides": {
+    "wc-sass-render": {
+      "sass": "$sass"
+    }
+  },
```

Comment:
What does this section of code do and why?

There is nothing in the PR description that explains why this is here.

Reply:
https://stackoverflow.com/questions/15806152/how-do-i-override-nested-npm-dependency-versions

wc-sass-render has a local dependency for an older version of Sass `"sass": "^1.62.1"`, with Sass installed locally we can then dictate that we use that version over the wc-sass-render dependency version.